src/abci/server.py

```diff
@@ -183,10 +183,12 @@ async def _handler(
             data.write(bits)
             data.seek(last_pos)
 
-            ## Tendermint prefixes each serialized protobuf message
-            ## with varint encoded length. We use the 'data' buffer to
-            ## keep track of where we are in the byte stream and progress
-            ## based on the length encoding
+            """
+            Tendermint prefixes each serialized protobuf message
+            with varint encoded length. We use the 'data' buffer to
+            keep track of where we are in the byte stream and progress
```

Comment:
docstrings > comments imo, docstrings have a runtime memory footprint ( can be accessed @ .__doc__ attribute )

Reply:
I've changed the docstring to a comment